pyposeidon/schism.py

```diff
@@ -1819,11 +1819,11 @@ def set_obs(self, **kwargs):
         stations.index += 1
         stations["gindex"] = mesh_index
         try:
```

Comment:
I would even suppress assigning `self.obs`  (above here line 1755: `self.obs = tg_database`) 
otherwise it gets written to `<module>_model.json`

Reply:
with the new implementation tg_database (and thus `self.obs`) is always a filename, either originally or after using `searvey`, 

```python
            tg_database = os.path.join(path, "stations.json")
            tg.to_file(tg_database)
        self.obs = tg_database
```